glean-core/android/src/main/java/mozilla/telemetry/glean/error/ErrorRecording.kt

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+@file:Suppress("MatchingDeclarationName")
+
+package mozilla.telemetry.glean.error
+
+import androidx.annotation.VisibleForTesting
+import mozilla.telemetry.glean.Dispatchers
+import mozilla.telemetry.glean.private.CounterMetricType
+import mozilla.telemetry.glean.private.Lifetime
+
+// Test-only internal object.
+//
+// This allows to check the number of errors recorded per metric.
+
+@VisibleForTesting(otherwise = VisibleForTesting.NONE)
+object ErrorRecording {
```

Comment:
Does this really need to be an `object`? Doesn't look like we're storing any persisted state, maybe we could just make this a `class` with a `companion object` or just have no class at all and just have `ErrorType`, `GLEAN_ERROR_NAMES` and `testGetNumRecordedErrors` be loose in the file. Can you file a bug about this?

Reply:
I don't mind this sticking around as a "namespace" class just for organization (it's the same way in glean-ac), but don't feel strongly either way.

Reply:
Right, that's true. I'd still file a bug, just for discussing this again "in the future".